apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx

```diff
@@ -4,22 +4,22 @@ const steps = [
   {
     element: "codemod",
     description:
-      "<p>This section is dedicated to codemods.</p><p>Codemods are tools used to automate code modifications and refactorings. They allow developers to make large-scale changes across a codebase quickly and consistently.</p><p> Here, you can see an example codemod script that transforms code from a 'Before' state to an 'After' state using jscodeshift.</p><p> You can modify the codemod's code here.</p>",
+      "<p>This section is dedicated to codemods.</p><p>Codemods are tools used to automate code modifications and refactorings. They allow developers to make large-scale changes across a codebase quickly and consistently.</p><p>The codemod editor allows you to see and modify the codemod you are building.</p>",
```

Comment:
too much text. folks who are here already know what codemod is.

what they need to know here is that this is the engine, which you are not talking about here.
make it super super concise... too many words here.

i can share my thoughts later but feel free to use your best judgement and ship. we iterate with user feedback.

Reply:
maybe in the intro we can also place some links to docs? Or even link the definition of codemod for folks who don't know the basics

Reply:
Link to generic docs should already be in the studio. Engine-specific guides are nice but make this feature complicated. Let's keep it simple for now we iterate with user feedback later.

Reply:
updated. Shortened and mentioned `jscodeshift` and `ts-morph` as the supported engines to use in studio. Please check and lmk if you like this more.